CHANGELOG.md

```diff
@@ -96,4 +101,4 @@ Changes: https://github.com/mapbox/mason/compare/v0.1.1...v0.2.0
  - Improved llvm >= 3.8.1 packages to support `-flto` on linux
 
 ## 0.1.0
- - First versioned release
\ No newline at end of file
+ - First versioned release
```

Comment:
Seem like the text editor used is adding them on purpose. In this case, you can e.g. use `git add -p` and avoid that particular change.